[2024-10-16T02:21:51.556+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-16T02:21:51.872+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_sells_directory manual__2024-10-16T02:21:41.973104+00:00 [queued]>
[2024-10-16T02:21:52.284+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_update_stock_ferrimac.extract_data_sells_directory manual__2024-10-16T02:21:41.973104+00:00 [queued]>
[2024-10-16T02:21:52.301+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-16T02:21:52.760+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data_sells_directory> on 2024-10-16 02:21:41.973104+00:00
[2024-10-16T02:21:52.859+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-16T02:21:52.901+0000] {standard_task_runner.py:72} INFO - Started process 104 to run task
[2024-10-16T02:21:52.953+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_update_stock_ferrimac', 'extract_data_sells_directory', 'manual__2024-10-16T02:21:41.973104+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/object_dag.py', '--cfg-path', '/tmp/tmpx0k1asur']
[2024-10-16T02:21:53.012+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask extract_data_sells_directory
[2024-10-16T02:21:53.524+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_update_stock_ferrimac.extract_data_sells_directory manual__2024-10-16T02:21:41.973104+00:00 [running]> on host 2296928c39b5
[2024-10-16T02:22:01.606+0000] {job.py:229} INFO - Heartbeat recovered after 10.91 seconds
[2024-10-16T02:22:18.962+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_update_stock_ferrimac' AIRFLOW_CTX_TASK_ID='extract_data_sells_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T02:21:41.973104+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T02:21:41.973104+00:00'
[2024-10-16T02:22:19.006+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-16T02:22:19.198+0000] {logging_mixin.py:190} INFO - The file 'ventas_unidades.xlsx' was not found in the sells directory.
[2024-10-16T02:22:19.199+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-16T02:22:19.306+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-16T02:22:19.318+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_update_stock_ferrimac, task_id=extract_data_sells_directory, run_id=manual__2024-10-16T02:21:41.973104+00:00, execution_date=20241016T022141, start_date=20241016T022151, end_date=20241016T022219
[2024-10-16T02:22:19.545+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-16T02:22:19.572+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-10-16T02:22:19.661+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
